src/metabase/driver/sql_jdbc/sync/interface.clj

```diff
@@ -127,6 +127,14 @@
   driver/dispatch-on-initialized-driver
   :hierarchy #'driver/hierarchy)
 
+(defmacro ^:private with-quoting [driver & body]
+  `(binding [sql/*dialect* (sql/get-dialect (sql.qp/quote-style ~driver))
+             sql/*quoted*  true]
+     ~@body))
```

Comment:
DRY?

Reply:
Typically I only DRY if one of the following is true:

- The usages are in the same namespace
- There are at least 3 instances of the duplication

Pulling the helper out to a third place doesn't seem worth it in this case, but I'm not hard against it if you feel strongly.